src/style.css

```diff
@@ -1 +1,34 @@
-/* add styles here */
+body {
+  margin: 0 auto;
+  display: flex;
+}
+
+.main-container {
+  flex-direction: column;
+  display: flex;
+  margin: 0 auto;
+  justify-content: space-between;
+}
+
+.stars {
+  display: flex;
+  margin-bottom: 0;
+}
+
+.stars__star {
+  width: 16px;
+  height: 16px;
+  background-image: url('./images/star.svg');
+  background-position: center;
+  background-repeat: no-repeat;
+  margin-right: 4px;
+}
+
+.stars--0 .stars__star:nth-child(-n + 0),
+.stars--1 .stars__star:nth-child(-n + 1),
+.stars--2 .stars__star:nth-child(-n + 2),
+.stars--3 .stars__star:nth-child(-n + 3),
+.stars--4 .stars__star:nth-child(-n + 4),
+.stars--5 .stars__star:nth-child(-n + 5) {
```

Comment:
The nth-child selectors are incorrect. For example, '.stars--0 .stars__star:nth-child(-n + 0)' will not select any stars because '-n + 0' results in zero, which is not a valid nth-child index. Consider adjusting these selectors to correctly highlight the intended number of stars for each '.stars--X' class.